eodag/resources/providers.yml

```diff
@@ -2175,9 +2176,6 @@
       expver:
         - expver
         - '$.expver'
-      fcmonth:
-        - fcmonth
-        - '$.fcmonth'
```

Comment:
why removing fcmonth and frame in the file ?

Reply:
The parameters actually do not exist for any of the product types.